scripts/refresh_leaderboard_counters.ts

```diff
@@ -1,50 +1,60 @@
 import 'dotenv/config';
-import { refreshProposalsCount, refreshVotesCount } from '../src/helpers/actions';
 import db from '../src/helpers/mysql';
+import { refreshProposalsCount, refreshVotesCount } from '../src/helpers/actions';
 
 // Usage: yarn ts-node scripts/refresh_leaderboard_counters.ts --space [OPTIONAL-SPACE-ID] --pivot TIMESTAMP
 async function main() {
-  const filters: string[] = [];
+  let spacesFilter = '';
+  let usersFilter = '';
 
   process.argv.forEach((arg, index) => {
     if (arg === '--space') {
       if (!process.argv[index + 1]) throw new Error('Space ID is missing');
-      console.log('Filtered by space:', process.argv[index + 1]);
-      filters.push(`id = '${process.argv[index + 1]}'`);
+      console.log('Filtered by spaces.id =', process.argv[index + 1]);
+      spacesFilter = `WHERE space = '${process.argv[index + 1].trim()}'`;
     }
 
     if (arg === '--pivot') {
       if (!process.argv[index + 1]) throw new Error('Pivot timestamp is missing');
-      console.log('Filtered by created >= ', process.argv[index + 1]);
-      filters.push(`created >= ${process.argv[index + 1]}`);
+      console.log('Filtered by users.created >=', process.argv[index + 1]);
+      usersFilter = `WHERE created >= ${process.argv[index + 1].trim()}`;
     }
   });
+  console.log('');
 
-  const filtersQuery = filters.length ? `WHERE ${filters.join(' AND ')}` : '';
-  const query = `SELECT id, name, created FROM spaces ${filtersQuery} ORDER BY created ASC`;
+  const users: { id: string; created: number }[] = await db.queryAsync(
+    `SELECT id, created FROM users ${usersFilter} ORDER BY created ASC`
```

Comment:
But not every voter/proposal creator is on `users`

Reply:
Ouch, was not aware of this

Reply:
Script updated to loop by voter instead